examples/ct/ctmapper/lookup/lookup.go

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// The lookup binary looks up a specific ID in a map.
```

Comment:
I believe `package main` is an exception and needs to take use the name of the resulting binary.